config/layouts.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Genesis Sample child theme.
+ *
+ * @author  StudioPress
+ * @license GPL-2.0-or-later
+ * @link    https://my.studiopress.com/themes/genesis-sample/
+ */
+
+// Pull default layouts from Genesis
+$genesis_config = genesis_get_config( 'layouts', 'secondary' );
```

Comment:
@nathanrice I'm currently reviewing the `genesis_get_config()` PR and it doesn't accept a second parameter.